cross/readarr/Makefile

```diff
@@ -0,0 +1,36 @@
+PKG_NAME = Readarr
+PKG_VERS = 0.1.0.1225
+PKG_EXT = tar.gz
+PKG_DIST_NAME = updatefile?os=linux\&runtime=netcore\&arch=$(PKG_DIST_ARCH)
+PKG_DIST_FILE = $(PKG_NAME).develop.$(PKG_VERS).linux-core-$(PKG_DIST_ARCH).$(PKG_EXT)
+PKG_DIST_SITE = http://readarr.servarr.com/v1/update/nightly
```

Comment:
Rather than nightly (I assume every day results in a new build?) is there a more stable version?

Reply:
`https://readarr.servarr.com/v1/update/develop/updatefile?os=linux&runtime=netcore&arch=x64` ?

Reply:
According to their [wiki](https://wiki.servarr.com/readarr/installation#linux): "Readarr is currently in beta testing and thus using the nightly branch."

Reply:
Readarr has been formally develop as of early last week